webapp/src/components/sidebar_header/sidebar_header.tsx

```diff
@@ -0,0 +1,25 @@
+import React, {PureComponent, ReactElement} from 'react';
+
+import {Theme} from 'mattermost-redux/types/preferences';
+
+import SidebarButtons from '../sidebar_buttons';
+
+interface SidebarHeaderProps{
+    show: boolean,
+    theme: Theme
+}
```

Comment:
Typically we would have the type like this to differentiate types from real data, though it's not configured in eslint for some reason

```suggestion
interface SidebarHeaderProps {
    show: boolean;
    theme: Theme;
}
```